node/http.ts

```diff
@@ -152,6 +154,35 @@ class ClientRequest extends NodeWritable {
     this.destroy();
   }
 
+  async _createBody(
+    body: ReadableStream | null,
+    opts: RequestOptions,
+  ): Promise<Buffer | ReadableStream | null> {
+    if (!body) return null;
+
+    const isChunked = (opts: RequestOptions) => {
```

Comment:
Can you either inline this code or move the function to the top level. The way this is currently written, it will need to unnecessarily create a new closure each time `_createBody()` is called.

Reply:
Thank you for pointing this out. I have inlined the code. (a29e9d9)